File: WebApp/Models/Organisation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApi.Models
{
    public class OrganisationDTO
    {
        public string Id { get; set; }

        public string Name { get; set; }

        public OrgUserDTO RootUser { get; set; }

        public string TelNumber { get; set; }

        public string AddressLine1 { get; set; }

        public string AddressLine2 { get; set; }

        public string Town { get; set; }

        public string County { get; set; }

        public string Postcode { get; set; }

        public Guid DefaultLanguageId { set; get; }

        public Guid DefaultCalendarId { set; get; }

        public bool SubscriptionEnabled { get; set; }

        public decimal? SubscriptionMonthlyRate { get; set; }
    }
}

File: WebApp/wwwroot/scripts/domain/organisation.ts
﻿module App.Models {
    "use strict";

    export interface IOrganisation extends ng.resource.IResource<IOrganisation> {
        id: string;
        name: string;
        rootUser: IOrgUser;
        telNumber: string;
        addressLine1: string;
        addressLine2: string;
        town: string;
        county: string;
        postcode: string;
        description: string;
        website: string;
        logoUrl: string;
        termsAndConditions: string;
        requiresAgreement: boolean;
        defaultLanguageId: string;
        defaultCalendarId: string;
        subscriptionEnabled: boolean;
        subscriptionMonthlyRate?: number;

        facebookId: string;
        twitterId: string;
        instagramId: string;
        skypeId: string;
        linkedinUrl: string;
        youTubeUrl: string;
    }
}